nuitka/plugins/PluginBase.py

```diff
@@ -910,3 +914,28 @@ def wrapped(*args, **kwargs):
                 return None
 
     return wrapped
+
+
+class TagContext(dict):
+    def __init__(self, full_name, *args, **kwargs):
+        self.full_name = full_name
+        super().__init__(*args, **kwargs)
```

Comment:
Since `super()` works badly for older Python, I avoid it always, just write `dict.__init__` and be done with it. Yes not pretty, but for portable Python you need to pass arguments, that petty much render it useless to use.